x/staking/keeper/grpc_query.go

```diff
@@ -649,3 +653,17 @@ func redelegationsToRedelegationResponses(ctx context.Context, k *Keeper, redels
 
 	return resp, nil
 }
+
+// setConsensusAddress sets the ConsensusAddress field for the given validator
+func setConsensusAddress(validator *types.Validator, consensusAddressCodec addresscodec.Codec) {
+	if validator == nil {
```

Comment:
Is there a scenario where this can be true?

Reply:
Honestly I don't know, I put there because I wanted to make it nil-safe

Reply:
@lucaslopezf Wouldn't it be better to raise a panic along with an error log if the validator is nil?"